### PR TITLE
chore: update dev deps

### PR DESCRIPTION
**Pull request by robjtede**
noticed this from https://deps.rs/crate/ed25519-dalek

### CODE REVIEW

File: curve25519-dalek/src/scalar.rs [hunk 1]
```diff
@@ -1952,13 +1952,13 @@ pub(crate) mod test {
...
         assert_eq!(
-            Scalar::ROOT_OF_UNITY.pow(&[1u64 << Scalar::S, 0, 0, 0]),
+            Scalar::ROOT_OF_UNITY.pow([1u64 << Scalar::S, 0, 0, 0]),
             Scalar::ONE,
         );
 
         // DELTA^{t} mod m == 1
         assert_eq!(
-            Scalar::DELTA.pow(&[
+            Scalar::DELTA.pow([
```
Comment (rozbb) [inline curve25519-dalek/src/scalar.rs:1961]: Are these changes a result of a dev dep update?

Comment (tarcieri) [reply curve25519-dalek/src/scalar.rs:1961]: Probably clippy complaining about a needless borrow after merging #562

Comment (rozbb) [reply curve25519-dalek/src/scalar.rs]: Ah ok

**Comment by rozbb** [comment]
Thank you!

**Pull request merged**